### ISSUE TITLE
Change the wording next to import of categories to make it more clear

### ISSUE THREAD

**Issue by vytisbulkevicius** [Codeinwp/feedzy-rss-feeds#868]
### What problem does this address?

Currently we have the field where we assign taxonomy looking like this:
![image](https://github.com/Codeinwp/feedzy-rss-feeds/assets/36594177/ab2ce9a7-53fb-4de9-9a75-ff5f0c1da919)

It's not very clear and we mention taxonomy on the left, category above the field itself and description is confusing as well.

We can rephrase it to (**read more opens in a new tab**):

> The imported post will be assigned to a selected taxonomy. Leave it blank if you don't need a taxonomy. 
You can automatically create categories with a magic tag **[#item_categories]** or use custom tag parsing [Read More](https://docs.themeisle.com/article/1154-how-to-use-feed-to-post-feature-in-feedzy#dynamic-post-taxonomy).

So with Free version of the plugin it should look like this:
![image](https://github.com/Codeinwp/feedzy-rss-feeds/assets/36594177/734ab3ad-dcf2-447c-a2d1-4d94f8cf3550)

With the PRO version:
<img width="1191" alt="image" src="https://github.com/Codeinwp/feedzy-rss-feeds/assets/36594177/77e4fec5-9602-4c3f-aad5-8bd9f9559b20">

### Will this feature require documentation? (Optional)

None

**Comment by girishpanchal30** [comment]
@vytisbulkevicius I've changed the text as per your suggestion with this PR https://github.com/Codeinwp/feedzy-rss-feeds/pull/869
